libs/hdf-converters/src/twistlock-mapper.ts

```diff
@@ -67,7 +68,7 @@ export class TwistlockMapper extends BaseConverter {
             key: 'id',
             tags: {
               nist: ['SI-2', 'RA-5'],
-              cci: ['CCI-002605', 'CCI-001643'],
+              cci: getCCIsForNISTTags(['SI-2', 'RA-5']),
```

Comment:
Don't we have a constant somewhere that defines our default nist tags?  Use that.